### DISCUSSION TITLE
My workflow with raw photos and editing - still some problems

### DISCUSSION THREAD

**Discussion by scotch666** [photoprism/photoprism#1375] [Q&A] [answered]
Hello, 
I'm trying to find a good workflow with photoprism, but was not really successful until now. Darktable is my favorite photo editing software and I would like to use it along with photoprism. Both, photoprism and darktable, are accessing the same directory, where the  originals are stored. Darktable is configured to save xmp files along the original photo (jpg or raw) in which the "photo editing settings" of darktable are saved.
Here are some of the problems I'm facing right now:
- When indexing the originals the first time, photoprism converts raw files with the darktable settings from the xmp as expected. After editing the raw file again with darktable, I haven't found any way to update the thumbnails in photoprism according to the new settings in the xmp. I'm searching for a way to reconvert the thumbnails of the edited images. Reindexing does not help. Even the option "full rescan" does not help, which would't be a real option, because it would take way too long. What I think would be nice, if photoprism looks for changes in the xmp file and if changes are found, the thumbnails are recreated.
- Photoprism does ignore xmp files for jps totally: It could also happen, that I edit jpgs with darktable. I would then expect photoprism to create the jpgs with darktable as well to have the correct thumbnails as I edited them in darktable. Right now, photoprism seems to create the thumbnails directly from the jpg and ignores the xmp-file.

Is there any solution to my problems? How is your workflow with photoprism and image editing?

BR
Xaver

**Comment by lastzero** [comment, marked as answer]
It would be easiest to render / create / update JPEGs manually (or automatically with Darktable) rather than expecting PhotoPrism to scan your XMP files for relevant changes and rerender JPEGs for you. We might build that feature at some point, but it's usually best to update JPEGs on your own if you edit photos anyway.
